### PR TITLE
Automatic port of System.Linq exceptions

### PR DESCRIPTION
**Pull request by carlossanlop**
Ported using [DocsPortingTool](https://github.com/carlossanlop/DocsPortingTool) with the `-PortMemberExceptions true` argument.

### CODE REVIEW

File: xml/System.Linq/ParallelEnumerable.xml [hunk 1]
```diff
@@ -624,7 +624,11 @@
   
  -or-  
   
- if AsOrdered is called midway through a query. It is allowed to be called immediately after <see cref="M:System.Linq.ParallelEnumerable.AsParallel(System.Collections.IEnumerable)" />, <see cref="M:System.Linq.ParallelEnumerable.Range(System.Int32,System.Int32)" /> or <see cref="M:System.Linq.ParallelEnumerable.Repeat``1(``0,System.Int32)" />.</exception>
+ if AsOrdered is called midway through a query. It is allowed to be called immediately after <see cref="M:System.Linq.ParallelEnumerable.AsParallel(System.Collections.IEnumerable)" />, <see cref="M:System.Linq.ParallelEnumerable.Range(System.Int32,System.Int32)" /> or <see cref="M:System.Linq.ParallelEnumerable.Repeat``1(``0,System.Int32)" />.
```
Comment (gewarren) [inline xml/System.Linq/ParallelEnumerable.xml:627]:
```suggestion
 if <see langword="AsOrdered" /> is called midway through a query. It is allowed to be called immediately after <see cref="M:System.Linq.ParallelEnumerable.AsParallel(System.Collections.IEnumerable)" />, <see cref="M:System.Linq.ParallelEnumerable.Range(System.Int32,System.Int32)" />, or <see cref="M:System.Linq.ParallelEnumerable.Repeat``1(``0,System.Int32)" />.
```

File: xml/System.Linq/ParallelEnumerable.xml [hunk 2]
```diff
@@ -624,7 +624,11 @@
...
+
+-or-
+
+<paramref name="source" /> is not one of <see cref="M:System.Linq.ParallelEnumerable.AsParallel(System.Collections.IEnumerable)" />, <see cref="M:System.Linq.ParallelEnumerable.Range(System.Int32,System.Int32)" /> or <see cref="M:System.Linq.ParallelEnumerable.Repeat``1(``0,System.Int32)" />.</exception>
```
Comment (gewarren) [inline xml/System.Linq/ParallelEnumerable.xml:631]:
```suggestion
<paramref name="source" /> is not one of <see cref="M:System.Linq.ParallelEnumerable.AsParallel(System.Collections.IEnumerable)" />, <see cref="M:System.Linq.ParallelEnumerable.Range(System.Int32,System.Int32)" />, or <see cref="M:System.Linq.ParallelEnumerable.Repeat``1(``0,System.Int32)" />.</exception>
```

File: xml/System.Linq/ParallelEnumerable.xml [hunk 3]
```diff
@@ -3621,7 +3625,7 @@
         <remarks>To be added.</remarks>
         <exception cref="T:System.OperationCanceledException">The query was canceled with the token passed in through <paramref name="WithCancellation" />.</exception>
         <exception cref="T:System.ArgumentNullException">
-          <paramref name="source" /> or <paramref name="action" /> is a null reference (Nothing in Visual Basic).</exception>
+          <paramref name="source" /> or <paramref name="action" /> or <paramref name="keySelector" /> or <paramref name="elementSelector" /> or <paramref name="resultSelector" /> is <see langword="null" /> (Nothing in Visual Basic).</exception>
```
Comment (gewarren) [inline xml/System.Linq/ParallelEnumerable.xml:3628]:
```suggestion
          <paramref name="source" /> or <paramref name="action" /> or <paramref name="keySelector" /> or <paramref name="elementSelector" /> or <paramref name="resultSelector" /> is <see langword="null" /> (<see langword="Nothing" /> in Visual Basic).</exception>
```

File: xml/System.Linq/ParallelEnumerable.xml [hunk 4]
```diff
@@ -3683,7 +3687,7 @@
...
```
Comment (gewarren) [inline xml/System.Linq/ParallelEnumerable.xml:3690]:
```suggestion
          <paramref name="source" /> or <paramref name="action" /> or <paramref name="keySelector" /> or <paramref name="elementSelector" /> or <paramref name="resultSelector" /> is <see langword="null" /> (<see langword="Nothing" /> in Visual Basic).</exception>
```

File: xml/System.Linq/ParallelEnumerable.xml [hunk 5]
```diff
@@ -3831,7 +3835,7 @@
         <remarks>To be added.</remarks>
         <exception cref="T:System.OperationCanceledException">The query was canceled with the token passed in through <paramref name="WithCancellation" />.</exception>
         <exception cref="T:System.ArgumentNullException">
-          <paramref name="source" /> or <paramref name="action" /> is a null reference (Nothing in Visual Basic).</exception>
+          <paramref name="source" /> or <paramref name="action" /> or <paramref name="outer" /> or <paramref name="inner" /> or <paramref name="outerKeySelector" /> or <paramref name="innerKeySelector" /> or <paramref name="resultSelector" /> is <see langword="null" /> (Nothing in Visual Basic).</exception>
```
Comment (gewarren) [inline xml/System.Linq/ParallelEnumerable.xml:3838]:
```suggestion
          <paramref name="source" /> or <paramref name="action" /> or <paramref name="outer" /> or <paramref name="inner" /> or <paramref name="outerKeySelector" /> or <paramref name="innerKeySelector" /> or <paramref name="resultSelector" /> is <see langword="null" /> (<see langword="Nothing" /> in Visual Basic).</exception>
```

File: xml/System.Linq/ParallelEnumerable.xml [hunk 6]
```diff
@@ -3969,7 +3973,7 @@
...
```
Comment (gewarren) [inline xml/System.Linq/ParallelEnumerable.xml:3976]:
```suggestion
          <paramref name="source" /> or <paramref name="action" /> or <paramref name="outer" /> or <paramref name="inner" /> or <paramref name="outerKeySelector" /> or <paramref name="innerKeySelector" /> or <paramref name="resultSelector" /> is <see langword="null" /> (<see langword="Nothing" /> in Visual Basic).</exception>
```

File: xml/System.Linq/ParallelEnumerable.xml [hunk 7]
```diff
@@ -4355,7 +4359,7 @@
...
```
Comment (gewarren) [inline xml/System.Linq/ParallelEnumerable.xml:4362]:
```suggestion
          <paramref name="source" /> or <paramref name="action" /> or <paramref name="outer" /> or <paramref name="inner" /> or <paramref name="outerKeySelector" /> or <paramref name="innerKeySelector" /> or <paramref name="resultSelector" /> is <see langword="null" /> (<see langword="Nothing" /> in Visual Basic).</exception>
```

File: xml/System.Linq/ParallelEnumerable.xml [hunk 8]
```diff
@@ -4493,7 +4497,7 @@
...
```
Comment (gewarren) [inline xml/System.Linq/ParallelEnumerable.xml:4500]:
```suggestion
          <paramref name="source" /> or <paramref name="action" /> or <paramref name="outer" /> or <paramref name="inner" /> or <paramref name="outerKeySelector" /> or <paramref name="innerKeySelector" /> or <paramref name="resultSelector" /> is <see langword="null" /> (<see langword="Nothing" /> in Visual Basic).</exception>
```

File: xml/System.Linq/ParallelEnumerable.xml (hunk 4 above)
Comment (carlossanlop) [reply xml/System.Linq/ParallelEnumerable.xml:3690]: Good catch. I think I can ask the tool to substitute it automatically (if the whole string is `Nothing in Visual Basic`).

**Pull request merged**